doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -475,12 +475,11 @@ Mixed references is the default to build when no options are specified. _Note th
 - `--with-mixedrefs=static` (this is the default) create a mixed references build which avoids runtime checks by compiling source twice
 - `--with-noncompressedrefs` create a build supporting non-compressed references only
 
-:pencil: **OpenSSL support:** If you want to build an OpenJDK that includes OpenSSL, you must specify `--with-openssl={fetched|system|path_to_library}`
```

Comment:
Why did you remove `system` from this list? It's still a supported choice.

Reply:
For Windows there is a check in the code. We don't have a check for Mac but I assumed it doesn't work on this platform either.
`printf "On Windows, value of \"system\" is currently not supported with --with-openssl. Please build OpenSSL using VisualStudio outside cygwin and specify the path with --with-openssl\n"`

Reply:
Sorry, I thought you had removed it from all platforms.